### PR TITLE
Update shadermodel.md

### PR DESCRIPTION
**Pull request by TylerDahl**
**What does this PR do?**
Update shadermodel documentation to replace auto-generated text.

Thanks for the contribution! Please provide a concise description of the problem this request solves.

**How does this PR change Premake's behavior?**
Modifies the Premake documentation.

Are there any breaking changes? Will any existing behavior change?
No.

**Anything else we should know?**
No.

Add any other context about your changes here.

**Did you check all the boxes?**

- [CHECK ] Focus on a single fix or feature; remove any unrelated formatting or code changes
- [CHECK ] Add unit tests showing fix or feature works; all tests pass
- [CHECK ] Mention any [related issues](https://github.com/premake/premake-core/issues) (put `closes #XXXX` in comment to auto-close issue when PR is merged)
- [CHECK ] Follow our [coding conventions](https://github.com/premake/premake-core/blob/master/CONTRIBUTING.md#coding-conventions)
- [CHECK ] Minimize the number of commits
- [ CHECK] Align [documentation](https://github.com/premake/premake-core/tree/master/website) to your changes

*You can now [support Premake on our OpenCollective](https://opencollective.com/premake). Your contributions help us spend more time responding to requests like these!*

### CODE REVIEW

File: website/docs/shadermodel.md
Comment (nickclark2016) [inline website/docs/shadermodel.md:4]: Let's use `shadermodel ("value")` to keep consistent with other docs - (e.g.) https://premake.github.io/docs/objdir/

**Review by nickclark2016 — CHANGES_REQUESTED** [review]
Appreciate you taking time to update the shader docs. Everything is relatively minor, just want to keep things consistent across the docs 👍

File: website/docs/shadermodel.md
Comment (TylerDahl) [reply website/docs/shadermodel.md:4]: I'll make the relevant changes as requested.  Hey no problem, I've never been good at documentation but just wanted to make these small fixes to make things a little more clear for the future.

Comment (nickclark2016) [reply website/docs/shadermodel.md:4]: Appreciate all help with docs. When you get those changes out, I'll rereview and merge them so we can get it out.

**Comment by tommitytom** [comment]
Is there an example of this shader stuff anywhere?  And does it only support HLSL?

**Comment by TylerDahl** [comment]
Yes it only supports HLSL.  There is an example usage included in this change.

**Comment by TylerDahl** [comment]
@nickclark2016 Could you please approve this PR?

**Comment by nickclark2016** [comment]
Just saw the message, the last several weeks have been hectic for me.  Still waiting on you to address that last comment, then I'll go ahead and merge.

File: website/docs/shadermodel.md
Comment (nickclark2016) [reply website/docs/shadermodel.md]: Just poking this for line 39 as well.

**Comment by KyrietS** [comment]
@TylerDahl ?

**Comment by TylerDahl** [comment]
@KyrietS Addressed the final issue for line 39.

**Comment by KyrietS** [comment]
Let's wait for @nickclark2016's approval

**Review by samsinsane — APPROVED** [review]

**Pull request merged**
